examples/medplum-demo-bots/src/deduplication/README.md

```diff
@@ -0,0 +1,56 @@
+# Deduplication Demo
+
+This repo contains code for deploying bots on [Deduplication](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#architecture-overview). 
+
+
+## Setup
+
+For deduplication - we have two bots, one for identifying a duplicated resource and creating a task, and the other that merges the two resources into one.
```

Comment:
Right idea, but worth expanding. A short description of each bot, what it does, and which step in that longer doc it corresponds to is also important, for people who get a link directly to this page. One bullet per bot.

Worth breaking out into its own h2 section